### PR TITLE
fix spacing

### PR DESCRIPTION
**Pull request by rettinghaus**
This removes unneeded whitespace.

### CODE REVIEW

File: resources/pedagogy.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you have questions or need assistance, please reach out to the [Pedagogy Inte
 ## Selected Bibliography
 
 <div class="columns">
-    <div class="column col-12 bibliography">
-        <script src="https://bibbase.org/show?bib=https%3A%2F%2Fapi.zotero.org%2Fgroups%2F4439204%2Fitems%3Fkey%3D8lnTbafaItBb4ZVOjRCEYliR%26format%3Dbibtex%26limit%3D100&jsonp=1&theme=dividers&nocache=1&authorFirst=1"></script>
-    </div>
+ <div class="column col-12 bibliography">
+ <script src="https://bibbase.org/show?bib=https%3A%2F%2Fapi.zotero.org%2Fgroups%2F4439204%2Fitems%3Fkey%3D8lnTbafaItBb4ZVOjRCEYliR%26format%3Dbibtex%26limit%3D100&jsonp=1&theme=dividers&nocache=1&authorFirst=1"></script>
```
Comment (musicEnfanthen) [inline resources/pedagogy.md:21]: `script` tag, as a child of `div.bibliography`, needs one more level of indentation

**Comment by musicEnfanthen** [comment]
Thank you @rettinghaus, looks good. 

off-topic: How did you find these space issues? (Sometimes it looks like a single white space is replaced by a single white space, but I guess this is an issue with the GitHub UI not making a difference between white spaces "encoded" differently in the markdown?)

**Comment by rettinghaus** [comment]
@musicEnfanthen Yes, these were special space characters like no-break spaces. Markdown linter complains about those.

**Comment by musicEnfanthen** [comment]
I see. Which linter do you run?

Many thanks for the fix and PR!

**Comment by rettinghaus** [comment]
@musicEnfanthen I'm using this one: https://github.com/DavidAnson/vscode-markdownlint

**Comment by bwbohl** [comment]
does this mean you replace no-break space characters with normal space characters?

**Comment by rettinghaus** [comment]
Yes, most likely.

**Pull request merged**